test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -72,8 +77,12 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 
 	ginkgo.When("Reconciling clusterQueue status", func() {
```

Comment:
```suggestion
	ginkgo.When("Reconciling clusterQueue usage in status", func() {
```

Then create a separate case just for the condition, with less flavors and test that the CQ is initially Active=false, then Active=true, and Active=false when you remove 1 flavor.